lms/djangoapps/api_manager/users/views.py

```diff
@@ -916,6 +916,35 @@ def delete(self, request, user_id, course_id):
         return Response({}, status=status.HTTP_204_NO_CONTENT)
 
 
+class UsersCoursesGradesList(SecureAPIView):
+    """
+    ### The UsersCoursesGradesList view allows clients to interact with the User's gradebook across courses
+    - URI: ```/api/users/{user_id}/courses/grades?courses={course_id1,course_id2}```
+    - GET: Returns a JSON representation of the matched set from the gradebook
+        * courses (optional): Set filtering parameter consisting of comma-separated course identifiers
+
+    ### Use Cases/Notes:
+    * Use the UsersCoursesGradesList view to interact with the User's gradebook across multiple Course enrollments
+    * Use GET to retrieve all of the Course gradebook entries for the specified User
+    """
+    def get(self, request, user_id):  # pylint: disable=unused-argument
+        """
+        GET /api/users/{user_id}/courses/grades
+        """
+        queryset = StudentGradebook.objects.filter(user=user_id)
+
+        response_data = []
+        for record in queryset:
+            response_data.append(
+                {
+                    'course_id': unicode(record.course_id),
+                    'current_grade': record.grade,
+                    'proforma_grade': record.proforma_grade,
+                }
```

Comment:
I've added the 'complete_status' flag -- take a look and let me know if it works for you.